src/Events/Integrations/Plugins/Event_Tickets/Emails/Email/Completed_Order.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Class Completed_Order.
+ *
+ * @since   TBD
```

Comment:
```suggestion
 * @since TBD
```
Let's do a search replace for this change.